x-pack/legacy/plugins/ml/server/models/job_validation/messages.js

```diff
@@ -252,14 +252,24 @@ export const getMessages = () => {
       }),
       url: 'https://www.elastic.co/guide/en/elasticsearch/reference/{{version}}/ml-job-resource.html#ml-job-resource'
     },
+    invalid_max_length: {
+      status: 'ERROR',
+      text: i18n.translate('xpack.ml.validation.messages.invalidMaxLengthErrorMessage', {
+        defaultMessage: 'The max length is {maxLength, plural, one {# character} other {# characters}}.',
+        values: {
+          maxLength: '{{maxLength}}',
+        },
+      }),
+      url: 'https://www.elastic.co/guide/en/elasticsearch/reference/{{version}}/ml-job-resource.html#ml-job-resource'
+    },
     job_id_valid: {
       status: 'SUCCESS',
       heading: i18n.translate('xpack.ml.models.jobValidation.messages.jobIdValidHeading', {
         defaultMessage: 'Job id format is valid',
       }),
       text: i18n.translate('xpack.ml.models.jobValidation.messages.jobIdValidMessage', {
         defaultMessage: 'Lowercase alphanumeric (a-z and 0-9) characters, hyphens or underscores, ' +
-          'starts and ends with an alphanumeric character.',
+          'starts and ends with an alphanumeric character, and is less than 64 characters.',
```

Comment:
Shouldn't this be less than `65 characters`? 64 chars is the maximum that is allowed.

Reply:
maybe "isn't longer than 64 characters."?